### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by dridk** [labsquare/CuteVCF#8]
I m not sure about the license ..  I m using htslib as a dynamics library . 
@Arkanosis Which opensource license can I use ?

**Comment by Arkanosis** [comment]
htslib is available under the MIT/Expat license and / or the modified 3-Clause BSD license, which means that you can basically use whichever license you want (including proprietary licenses), granted that you ship the content of htslib's [LICENSE](https://github.com/samtools/htslib/blob/aaf99a925850d0714701b88ee3ceab88881bab2e/LICENSE) file with you application.

The simplest way to proceed is probably to put the license you chose, together with the content of the aforementioned LICENSE file in your project's own LICENSE file, and to ship this file together with the binary releases.

The use of Qt, however, may restrict the license you can use somehow. The opensource license of Qt allows you to use most of its components under the terms of the LGPLv3 license, which means that as long as you link with Qt _dynamically_, you can still use whichever license you want (including proprietary licenses) for you own code. Some components, though, may only be used under the terms of the much more restrictive GPLv2/GPLv3 licenses, which means that however you link with them, you _have_ to use the GPLv2/GPLv3 license for your own code. These components include Qt Charts and Qt Data Visualisation.

To sum up: as long as you only link to htslib and the base Qt libraries _dynamically_, you can chose whichever license you want. But as soon as you use Qt Charts or Qt Data Visualisation, you _have_ to use either the GPLv2 or the GPLv3 license. Beware of additional dependencies which may have their own terms.

Hope this helps.
